lib/slim-rails/register_engine.rb

```diff
@@ -28,13 +28,10 @@ def self._register_engine(config)
         return unless config.respond_to?(:assets)
 
         config.assets.configure do |env|
-          if env.respond_to?(:register_transformer)
+          if env.respond_to?(:register_transformer) && Sprockets::VERSION.to_i == 4
```

Comment:
Does sprockets < 4 have `register_transformer`? Anyway, if this check is required, I hope `> 3` is better here as it would work for future versions.